TUTORIAL.md

```diff
@@ -491,7 +518,7 @@ node('remote') {
 }
```

Comment:
Right, sorry. Anyway this comment was misplaced—I meant to put it elsewhere on the file.